src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```
@@ -182,6 +182,11 @@ apiv2_recrutiment_candidate_history:
     _api: OrangeHRM\Recruitment\Api\CandidateHistoryAPI
     _key: candidateId
 
+apiv2_vacancy_list:
+  path: /api/v2/recruitment/fetch-vacancy-list
```

```suggestion
apiv2_recrutiment_vacancy_list:
  path: /api/v2/recruitment/public/vacancies
```

please read https://swagger.io/resources/articles/best-practices-in-api-design/

---

Done